Gemfile

```diff
@@ -37,6 +37,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rubocop', '~> 0.72.0', require: false
+  gem 'reek', '~> 5.4.0'
```

Comment:
why do we require one and not the other? (just asking)

Reply:
I wasn't aware of the meaning of `require: false` , I saw that was the suggested configuration of rubocop and I just applied it. Now that i'm aware of the meaning I think we should require it, do u agree ?

Reply:
I don't think we should require these since they are not needed when the app loads but when they analyze the code in the task. So I would try to not add them since they might make the code slower but we need to test if they are needed or not

Reply:
Done, as a convention we will not require gems unless it's strictly necessary.